src/components/flightPlan/flightPlan.js

```diff
@@ -0,0 +1,29 @@
+//import styled from "styled-components";
+import { MapContainer, TileLayer } from "react-leaflet";
+import { useNavigate } from "react-router-dom";
+
+export default function FlightPlan() {
+  const position = [53.633354, 9.999303];
+  const navigate = useNavigate();
+  return (
+    <div>
```

Comment:
maybe you could change the 'div' to a 'section' or 'article' to make it even more meaningful?